Framework/TestHelpers/src/ComponentCreationHelper.cpp

```diff
@@ -651,7 +651,62 @@ Instrument_sptr createTestInstrumentRectangular2(int num_banks, int pixels,
 }
 
 /**
- * createOneDetectorInstrument, creates the most simple possible definition of
+ * Create an test instrument with multiple nameless banks. Used for testing
+ * behaviour of name handling implementation in saveInstrument
+ *
+ * Banks are centered at (1*banknum, 0, 0) and are facing 0,0.
+ * Pixels are 4 mm wide.
+ *
+ * @param num_banks: number of rectangular banks to create
+ * @param pixels :: number of pixels in each direction.
+ * @param pixelSpacing :: padding between pixels
+ */
+Instrument_sptr createTestUnnamedRectangular2(int num_banks, int pixels,
```

Comment:
If this is largey the same as the existing method, can't we just add parameters to the original function to control the behaviour we want around the naming? Otherwise looks like a lot of copy-paste.

Reply:
done

Reply:
also /= comment done